CHANGELOG.md

```diff
@@ -2,13 +2,177 @@
 
 ## 10.0.0
 
-Teleport 10.0 is a major release of Teleport that contains new features, improvements, and bug fixes.
+Teleport 10 is a major release that brings:
+
+* Passwordless Preview
+* Resource Access Requests Preview
+* Proxy Peering Preview
+* IP-Based Restrictions Preview
+* Automatic User Provisioning Preview
+* Audit Logging for Microsoft SQL Server Database Access
+* Snowflake Database Access Preview
+* ElastiCache/MemoryDB Database Access Preview
+* Teleport Connect for Server and Database Access Preview
+* Machine ID Database Access Support Preview
+
+### Passwordless Preview
+
+Teleport 10 introduces passwordless support to your clusters. Any cluster with
+WebAuthn configured automatically benefits from passwordless. To use
+passwordless users may register a security key with resident credentials or use
+a built-in authenticator, like Touch ID.
+
+See https://goteleport.com/docs/access-controls/guides/passwordless/.
+
+### Resource Access Requests Preview
+
+Teleport 10 expands just-in-time access requests to allow for requesting access
+to specific resources, allowing users to request the least privileged access
+needed for their workflows.
+
+Just in time access requests are an enterprise-only feature.
+
+### Proxy Peering Preview
+
+Teleport 9 and prior required reverse tunnel agents to maintain connections to
+every proxy in the cluster. Teleport 10 introduces a new proxy peering mode that
```

Comment:
I think we should use a more specific term than "proxy" in phrases like "every proxy in the cluster", as our docs sometimes use this to indicate the Proxy Service, an SNI/ALPN client proxy, or a non-Teleport proxy deployed on the user's infrastructure. Maybe "Proxy Service instance"?